tests/CMakeLists.txt

```diff
@@ -72,16 +77,16 @@ set(CHECK_CHECK_EXPORT_HEADERS check_check.h)
 add_executable(check_check_export
   ${CHECK_CHECK_EXPORT_HEADERS}
   ${CHECK_CHECK_EXPORT_SOURCES})
-target_link_libraries(check_check_export check compat)
```

Comment:
Cleaning this up is ok. Can it be in a separate commit, though, as it is not directly related to adding support for regex?